modules/perception/camera/app/cipv_camera.cc

```diff
@@ -764,7 +773,7 @@ bool Cipv::DetermineCipv(const std::vector<base::LaneLine> &lane_objects,
       AINFO << "objects[" << i << "]->track_id: " << (*objects)[i]->track_id;
     }
     if (IsObjectInTheLane((*objects)[i], egolane_image, egolane_ground,
-                          &distance)) {
+                          world2camera, &distance) == true) {
```

Comment:
`== true` is unnecessary.

Reply:
Thank you for the review! I removed "== true" in other places also.